src/main/kotlin/me/bristermitten/devdenbot/extensions/Messages.kt

```diff
@@ -12,6 +15,29 @@ fun CommandEvent.arguments(): Arguments {
         , args.filter { !it.isFlag }, args.filter { it.isFlag })
 }
 
+suspend fun Message.getMentionedMember(index: Int, action: suspend (Member?) -> Unit = {}) {
```

Comment:
Why use a callback rather than just returning the Member?

Reply:
fixed